### PR TITLE
{2023.06}[system] EasyBuild v4.9.3

### PR DESCRIPTION
**Pull request by bedroge**

### CODE REVIEW

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+easyconfigs:
+  - EasyBuild-4.9.3.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/21412
+        from-commit: https://github.com/easybuilders/easybuild-easyconfigs/pull/21412
```
Comment (ocaisa) [inline easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml:5]:
```suggestion
        from-commit: 1cdd81524c974a29825e37bcf8ef3ccc291f5227
```

Comment (bedroge) [reply easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.2-001-system.yml:5]: Oops, copy-paste on my laptop is somewhat broken after upgrading to Ubuntu 24.04 :sweat_smile:

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic

**Comment by bedroge** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen4
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512

**Comment by boegel** [comment]
bot: build repo:eessi.io-2023.06-software arch:aarch64/a64fx

**Review by ocaisa — APPROVED** [review]

**Pull request merged**